5a/docker-compose-2Silos-1Client.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.1'
+
+services:
+  orleans-client:
+    image: orleans-client
+    build:
+      context: .
+      dockerfile: ./ops/Client/Dockerfile
+    environment:
+      - ADVERTISEDIP=172.17.0.1
```

Comment:
This is the default bridge network IP - you can find it out by running the command `docker network inspect bridge`

![image](https://user-images.githubusercontent.com/20638889/157853328-29bc2212-9aab-4caa-9b97-41de89ae5698.png)